draft-irtf-cfrg-hpke.md

```diff
@@ -379,17 +393,25 @@ following operations:
 
 - GenerateKeyPair(): Generate an ephemeral key pair `(skX, pkX)`
   for the DH group in use
+- DeriveKeyPair(ikm): Derive a key pair `(skX, pkX)` from
+  the byte string `ikm`, where `ikm` SHOULD have at least `Nsk` bytes
+  of entropy (see {{derive-key-pair}} for discussion)
 - DH(sk, pk): Perform a non-interactive DH exchange using the
   private key sk and public key pk to produce a Diffie-Hellman
   shared secret of length Ndh
 - Serialize(pk): Produce a byte string of length `Npk`
-  encoding the public key `pk`
+  encoding the public key `pk`.
 - Deserialize(enc): Parse a byte string of length `Npk` to recover a
   public key (note: this function can raise an error upon `enc` deserialization failure)
 - Ndh: The length in bytes of a Diffie-Hellman shared secret produced
   by the DH function of this KEM
 - Nsk: The length in bytes of a Diffie-Hellman private key
 
+`Serialize` implements both `SerializePublicKey` and `SerializeCiphertext`,
+as an encapsulated key is a Diffie-Hellman public key in this KEM
+algorithm. Likewise, `Deserialize` implements both `DeserializePublicKey`
+and `DeserializeCiphertext`. `Npk` and `Nenc` are equal.
```

Comment:
```suggestion
Since an encapsulated key is a Diffie-Hellman public key in this KEM algorithm,
`SerializePublicKey` and `SerializeCiphertext` are equivalent (`Npk` = `Nenc`).
Thus, we implement both using a single function called `Serialize`. `Deserialize`
is implemented similarly.
```

Reply:
:+1: